scRNA-seq/setup/mito_gene_lists.Rmd

```diff
@@ -97,13 +107,13 @@ mitogenes_hs %>%
 The procedure here is the same, starting with looking for mouse annotation files in AnnotationHub:
 
 ```{r}
-AnnotationHub::query(ah, c("Ensdb", "Mus musculus"))
+AnnotationHub::query(hub, c("Ensdb", "Mus musculus", "95"))
 ```
 
-Again, we will pick Ensembl 99, but we will skip saving the whole db  as its own object in the environment this time (it will still be cached).
+Again, we will pick Ensembl 95, but we will skip saving the whole db  as its own object in the environment this time (it will still be cached).
```

Comment:
```suggestion
Again, we will pick Ensembl 95, but we will skip saving the whole db as its own object in the environment this time (it will still be cached).
```